client/hotrod-client/src/test/java/org/infinispan/client/hotrod/HitsAwareCacheManagersTest.java

```diff
@@ -198,13 +200,15 @@ protected void destroy() {
     */
    public static class HitCountInterceptor extends BaseAsyncInterceptor {
       private static final Log log = LogFactory.getLog(HitCountInterceptor.class);
+      private static final Predicate<VisitableCommand> ignoreCommands = c ->
```

Comment:
Actually looking closer, this should be cleaner to just override the `visitEntrySetcommand` and just invoke `invokeNext` in that method.